content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml

```diff
@@ -128,6 +128,15 @@
                                                 value="h6"/>
                                         </items>
                                     </type>
+                                    <disableLink
```

Comment:
Added disable link checkbox under styles tab for now because another one of my teammate is working on a unified tab.